isobus/src/can_network_manager.cpp

```diff
@@ -267,35 +284,28 @@ namespace isobus
 	bool CANNetworkManager::add_protocol_parameter_group_number_callback(std::uint32_t parameterGroupNumber, CANLibCallback callback, void *parentPointer)
 	{
 		bool retVal = false;
-		CANLibProtocolPGNCallbackInfo callbackInfo;
+		CANLibProtocolPGNCallbackInfo callbackInfo{
+			callback, parentPointer, parameterGroupNumber
+		};
```

Comment:
Functionally the same as `ParameterGroupNumberCallbackData(parameterGroupNumber, callback, parent)`? which is used in all the other cases. Perhaps it would be better to switch this one as well.